crates/uv-cli/src/lib.rs

```diff
@@ -3271,6 +3271,10 @@ pub struct ToolInstallArgs {
     #[arg(long)]
     pub with: Vec<String>,
 
+    /// Additionally include entrypoints from the following packages.
+    #[arg(long)]
+    pub i_want_ponies: Vec<String>,
```

Comment:
Nit: Should be `Vec<PackageName>`

Reply:
Just double-checking: this field is the CLI args, other CLI fields also seems to be simpler type. Were you maybe suggesting to make this change on the `ToolInstallSettings` field? I think it makes sense to turn that one into a `BTreeSet<PackageName>` maybe, so that it is both de-duplicated and strongly typed.

Reply:
We use `PackageName` in the CLI where we can e.g. https://github.com/astral-sh/uv/blob/63b60bc0c8425f4475adbd0ba207159694946a08/crates/uv-cli/src/lib.rs#L1878

I'm all for converting it to a `BTreeSet` later too.